setup.cfg

```diff
@@ -91,7 +91,7 @@ profile=black
 known_first_party=tpi,tests
 
 [tool:pytest]
-timeout=5
+timeout=10
```

Comment:
Because the new `state_mv` method use a real terraform backend for the operation, not a mocked one, 5s might be too short for it.

> 3.73s call     tests/test_terraform.py::test_state_mv

Reply:
```suggestion
timeout=5
```